src/api/controllers/articleController.js

```diff
@@ -53,6 +52,110 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * publish a drafted article
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async publish(req, res) {
```

Comment:
According to the previous comment, I think this name should change... We publish articles that are ready to be viewed by the user and store draft until there ready to be published @alainburindi Please correct me if this is not the right flow of ideas.

Reply:
as you can see, this method changes the status of an article to published

Reply:
@alainburindi users should have the option of either publishing an article immediately or saving as a draft first.

Forcing them to store all articles as drafts then later go through another step to publish isn't great UX.